internal/ociartifact/store.go

```diff
@@ -479,35 +483,113 @@ func (s *Store) BlobMountPaths(ctx context.Context, artifact *Artifact, sys *typ
 		return nil, fmt.Errorf("failed to get an image reference: %w", err)
 	}
 
+	extractDir := filepath.Join(s.rootPath, "extracted", artifact.Digest().Encoded())
```

Comment:
unsure about this. is this a convention? is there a convention? seems odd to hardcode the path but I'm unsure about the alternative

Reply:
Yeah, I tried changing that logic. Let me know how it feels.https://github.com/cri-o/cri-o/pull/9150/commits/3c63062658829d4e028b53c107c45dc38434e639